examples/fabric/build_your_own_trainer/run.py

```diff
@@ -58,8 +58,8 @@ def train(model):
     from torchvision.datasets import MNIST
     from torchvision.transforms import ToTensor
 
-    train_set = MNIST(root="/tmp/data/MNIST", train=True, transform=ToTensor(), download=True)
-    val_set = MNIST(root="/tmp/data/MNIST", train=False, transform=ToTensor(), download=False)
+    train_set = MNIST(root="/tmp/data/MNIST", train=True, transform=ToTensor(), download=True)  # noqa: S108
+    val_set = MNIST(root="/tmp/data/MNIST", train=False, transform=ToTensor(), download=False)  # noqa: S108
```

Comment:
We need to add an exception rule for the examples, these shouldn't have the noqa notes